gatewayconfig/bluetooth/characteristics/diagnostics_characteristic.py

```diff
@@ -25,19 +26,58 @@ def __init__(self, service, eth0_mac_address, wlan0_mac_address):
                 ["read"], service)
         self.add_descriptor(DiagnosticsDescriptor(self))
         self.add_descriptor(OpaqueStructureDescriptor(self))
-        self.p2pstatus = False
-        self.eth0_mac_address = eth0_mac_address
-        self.wlan0_mac_address = wlan0_mac_address
+
+        firmware_version = os.getenv('FIRMWARE_VERSION')
```

Comment:
Just noticed this didn't get refactored originally. Can you populate this from args passed to app constructor instead? https://github.com/NebraLtd/hm-config/blob/e0c6f888c5205c4e6dbe24d0ca07b98654736426/gatewayconfig/gatewayconfig_app.py#L31

Reply:
@marvinmarnold I just refactored origininally. https://github.com/NebraLtd/hm-config/blob/ece48f0cf0ede2e0c4c3712035b32ea41e4ff135/gatewayconfig/bluetooth/characteristics/diagnostics_characteristic.py#L81
However, it makes more sense to get the `firmware_version` from app constructor.